package-files/systemd/infra.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
```

Comment:
Should we make this service `infra-connector`, instead of just `infra`? That would give us room to run other infra services.

Reply:
I think this is what we want. The trend has been to name connectors generally as `infra`